docs/README.md

```diff
@@ -0,0 +1,9 @@
+# How to update ONNX Runtime docs
+
+The docs for ONNX Runtime are published at https://onnxruntime.ai/docs.
```

Comment:
some of the docs still exist in the docs/ directory in other branches (e.g. master). what's the difference between those and "the docs" mentioned here?

Reply:
Docs in the repo should be more for technical reference, for developers of ORT source code. All customer-facing documentation content should eventually be moved to the gh-pages page.

Reply:
thanks for the clarification. that may be good to mention here